internal/service/imagebuilder/container_recipe.go

```diff
@@ -23,7 +23,7 @@ func ResourceContainerRecipe() *schema.Resource {
 		Update: resourceContainerRecipeUpdate,
 		Delete: resourceContainerRecipeDelete,
 		Importer: &schema.ResourceImporter{
-			State: schema.ImportStatePassthrough,
+			StateContext: schema.ImportStatePassthroughContext,
```

Comment:
Is this still required? https://www.terraform.io/plugin/sdkv2/resources/import#importer-state-function prefers the previous version?